components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/fee-mechanism.adoc

```diff
@@ -50,12 +50,20 @@ Let's go over a concrete example first. For example, imagine the following trace
 
 |===
 
-The proof will be closed and sent to L1 when any of these components becomes full. It's important to realize that the division to builtins must be predetermined! We can't decide on the fly to have proof with 20,000,001 Pedersen and nothing else.
+The proof is closed and sent to L1 when any of these components becomes full.
+It's important to realize that the division to builtins must be predetermined. We can't decide on the fly to have proof with 20,000,001 Pedersen and nothing else.
```

Comment:
```suggestion
```
The IMPORTANT below replaces this text, and is written more clearly, so we can get rid of it here.